File: api/v2010/recording/read-example-4/read-example-4.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

recordings = @client.recordings.list(
                                  date_created_before: Date.new(2016, 10, 15),
                                  date_created_after: Date.new(2016, 10, 12)
                                )

recordings.each do |record|
  puts record.sid
end

File: api/v2010/recording/read-example-4/read-example-4.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var recordings = RecordingResource.Read(
            dateCreatedBefore: new DateTime(2016, 10, 15, 0, 0, 0),
            dateCreatedAfter: new DateTime(2016, 10, 12, 0, 0, 0)
        );

        foreach(var record in recordings)
        {
           Console.WriteLine(record.Sid);
        }
    }
}
